### DISCUSSION TITLE
Add quarto as an app option

### DISCUSSION THREAD

**Discussion by trdunsworth** [dat-linux/community#21] [App, Extra, Book suggestions]
Could you add Posit's quarto application as an option to assist in creating reproducible output in HTML, PDF, or with pandoc installed, Microsoft-compatible Word documents? It's free, compatible with Python, R, Jupyter Notebooks, and Julia.

**Comment by dat-linux** [comment]
Yes, good idea. I might add it to the "extras" section as a convenience.

**Reply by trdunsworth** [reply]
I just did an update and found that it was there in the extras section!! Thank you!

**Reply by dat-linux** [reply]
Please let us know if you find any issues with interacting with the quarto program from whatever front ends you use it with. I can't imagine there being a problem, it's just an executable on the system path.

**Comment by trdunsworth** [comment]
I've already tried it and it works like a charm!! Thank you again!

On Thu, 23 Feb 2023 at 11:28, DAT Linux ***@***.***> wrote:

> Please let us know if you find any issues with interacting with the quarto
> program from whatever front ends you use it with. I can't imagine there
> being a problem, it's just an executable on the system path.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dat-linux/community/discussions/21#discussioncomment-5090481>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAOBEUPA73ZU5W75UUWJADWY6FZDANCNFSM6AAAAAAVBIRH6Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
"A mind needs books as a sword needs a whetstone, if it is to keep its
edge."
- George R. R. Martin
